### ISSUE TITLE
Frame cap doesn't appear to be working?

### ISSUE THREAD

**Issue by JNolan93** [PARTYMANX/partymod-thps4#13]
I notice that when I play the unmodded version, my fps counter seems accurate, it's a 120hz VRR display and when I limit the frames using Nvidia Control Panel, it runs at under 60hz when the game is being played. However, when I play the modded version, it stays at 118hz no matter what I do, I have the frame cap enabled in the .ini file, I've even tried limiting the frames via the Nvidia Control Panel and RTSS, nothing seems to work.

**Comment by PARTYMANX** [comment]
Just to confirm, you have `DisableFramerateCap=0` in the ini?  Otherwise, do you have the game installed into program files?  could it be one of those run as admin type of issues?  It shouldn't be possible for the cap to fail on some machines only. worst that could happen is a busy wait between frames.

Another thing to check, do you have the widescreen fixes mod installed?  maybe things are conflicting there

**Comment by JNolan93** [comment]
Yes, DissbleFramerateCap is set to 0. No, it's not installed in Program Files, it's on my dedicated gaming drive. I also don't have any widescreen fixes installed, except the ones your mod offers. I installed your mod on a clean install of the game. I'll try to play around with some settings, it's strange that it's not working for some reason.

**Comment by PARTYMANX** [comment]
yeah, that's bizarre.  and to be clear, you're running the game from THPS4.exe, right?  and the other patches, like controller input, work correctly I presume

**Comment by JNolan93** [comment]
Yes I'm running the game from THPS4.exe, and yes everything else seems to be working fine. It definitely is bizarre, even moreso because even trying to cap the frame rate in Nvidia Control Panel and RTSS doesn't work either. When I launch the original Skate4.exe, those programs work just fine limiting the frames rate. For shits and giggles I deleted the original exe, renamed the patched one to Skate4.exe to see if I could trick it lol, it didn't work and I didn't think it would, but I didn't know what else to try. There might be something conflicting with something else.
